docs/index.md

```diff
@@ -51,6 +51,7 @@ _**We only give support for the latest release/dev builds; please update before
 * [MCSF (My Christian Swear Filter)](https://www.spigotmc.org/resources/mcsf.54115/)
 * [MZP-VoteParty](https://www.spigotmc.org/resources/mzp-voteparty.89754/)
 * [InteractiveChat DiscordSRV Addon](https://www.spigotmc.org/resources/interactivechat-discordsrv-addon.83917/)
+* [DiscordSRVUtils Addon](https://discordsrvutils.xyz/)
```

Comment:
```suggestion
* [DiscordSRVUtils Addon](https://www.spigotmc.org/resources/discordsrvutils.85958/)
```
The website is currently down. I don't trust it to be up 24/7 so I'd appreciate if the url was replaced with the spigot resource page.